<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>181</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>449</width>
       <height>179</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>851</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">.QWidget{
background-color: rgba(29,181,225,0.2);
border-style: outset;
border-radius: 20;
border-color: rgba(29,181,225,0.15);
border-width: 7px;
}</string>
    </property>
    <widget class="QToolButton" name="zoomButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Zoom in/out</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/magnifier.png"/>
     </property>
    </widget>
    <widget class="QPushButton" name="fitToWindow">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Fit to window</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/full-size.png"/>
     </property>
    </widget>
    <widget class="QPushButton" name="normal">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Back to normal size</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/normal.png"/>
     </property>
    </widget>
    <widget class="QPushButton" name="select">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Select</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/selection-box.png"/>
     </property>
    </widget>
    <widget class="QDial" name="rotateDial">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Rotate</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>reset</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/restore.png"/>
     </property>
    </widget>
    <widget class="QPushButton" name="crop">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Crop image</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/crop-tool.png"/>
     </property>
    </widget>
    <widget class="QPushButton" name="undoButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Undo</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/undo.png"/>
     </property>
    </widget>
    <widget class="QPushButton" name="redoButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Redo</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border:1px solid transparent;
border-radius: 30;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme=":/resource/img/redo.png"/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(83,83,83);*/
/*color: black;*/
font-size:20px;
margin: 0;
padding: 0;
QAction::hover{
background-color:white;
color:  rgb(83,83,83) ;
}

</string>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>85</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83,83,83);
color: white;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
